File: Day 1/7/find_odd_or_even.c
#include <stdio.h>

const char* findOddOrEven(int n) {
    return (n % 2 == 0) ? "Even" : "Odd";
}

int main() {
    int n;

    printf("Enter an integer: ");
    scanf("%d", &n);

    printf("%s\n", findOddOrEven(n));
    return 0;
}

File: Day 1/7/findOddOrEven.js
const findOddOrEven = (n) => {
    return n % 2 === 0 ? "Even" : "Odd";
}

const findOddOrEvenAlgorithmically = (n) => {
    return (n & 1) === 0 ? "Even" : "Odd";
}

console.log(findOddOrEven(4));
console.log(findOddOrEvenAlgorithmically(4));